client/components/single-order/SingleOrderView.js

```diff
@@ -1,23 +1,63 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { createGetOrderThunk } from '../../store';
-import { Grid, Card, Table, Button, List } from 'semantic-ui-react';
+import {
+  createGetOrderThunk,
+  createGetCrewThunk,
+  createUpdateOrderThunk,
+} from '../../store';
+import {
+  Grid,
+  Card,
+  Table,
+  Button,
+  List,
+  Dropdown,
+  Container,
+} from 'semantic-ui-react';
 import PotholeRow from './PotholeRow';
 import moment from 'moment';
 
+const options = [
```

Comment:
This shouldn't be hardcoded. What happens if we add (or remove) a team tomorrow? Instead of grabbing a single crew, you should add all crews to the store on the mounting of this component. Then you don't have to have an asynchronous call to get the crew info in your handleClick()